src/pages/Categories/Categories.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Categories = () => (
+  <div>Check Status</div>
```

Comment:
- [ ] According to the requirements, the categories component should have a `button` with the text `check status`. Kindly change the `div` you used to a `button`

Reply:
Thank you for the reminder